AppSearchSample/app/src/main/java/com/android/example/appsearchsample/NoteAppSearchManager.kt

```diff
@@ -94,18 +95,16 @@ class NoteAppSearchManager(context: Context, coroutineScope: CoroutineScope) {
    * they were created (with most recent first). This returns a maximum of 10
    * documents that match the query, per AppSearch default page size.
    */
-  suspend fun queryLatestNotes(query: String): List<Note> {
+  suspend fun queryLatestNotes(query: String): List<SearchResult> {
     awaitInitialization()
 
     val searchSpec = SearchSpec.Builder()
       .setRankingStrategy(RANKING_STRATEGY_CREATION_TIMESTAMP)
+      .setSnippetCount(10)
```

Comment:
Please update Kdoc to mention this default as this may lead to an unexpected behavior if a user expects more than 10 query matches to be bolded, but only the first 10 are.